### DISCUSSION TITLE
Can Orcasound imagine a buoy-based low-cost live-listening hydrophone system for all orca signals?

### DISCUSSION THREAD

**Discussion by scottveirs** [orcasound/orcanode#59] [Ideas]
Let's discuss a buoyed solution. Mostly we have been using [cabled deployment strategies](https://github.com/orcasound/orcanode/wiki#orcasound-deployment-strategies), but in some otherwise-promising locations a low-cost offshore (probably buoyed**) hydrophone system would be handy...

**Pointed questions:**
_AKA preliminary design constraints_ for awesome design/engineer groups like the [Bainbridge BARN ](https://bainbridgebarn.org/)

- [ ] 1. Can new battery technology help us not need to change the batteries too often?
- [ ] 2. What current wireless technology could get the signal to shore (and ultimately the Orcasound live-listening app)?
- [ ] 3. What hydrophone deployment techniques will minimize flow, cable, and other sources of system or buoy/mooring noise?
- [ ] 4. What hydrophone and hardware combination would let us sense all signals of Southern Resident Killer Whales (calls, whistles, and clicks), or at least infer their presence by detecting calls (with mean fundamentals from 200-2000 Hz and harmonics up to ~20,000 Hz).

** Orcasound is totally open to other deployment strategies, but we are assuming for now that the tides and narrow channels of the Salish Sea rule out other extant approaches for near/continuous PAM, like [wave-gliders](https://www.liquid-robotics.com/wave-glider/how-it-works/), [autonomous boats](https://www.openoceanrobotics.com/), [sea gliders](https://apl.uw.edu/project/project.php?id=seaglider), etc.

**Comment by veirs** [comment]
Hydrophone Links: 
(see this repository's wiki: Hardware and field deployment -- for much information on our live streaming notions.)
[https://github.com/orcasound/orcanode/wiki](https://github.com/orcasound/orcanode/wiki)

[https://www.instructables.com/Lets-Build-Some-World-Class-Hydrophones/](https://www.instructables.com/Lets-Build-Some-World-Class-Hydrophones/)

Bioacoustice group in StackExchange:
[https://bioacoustics.stackexchange.com/questions/1269/what-affordable-hydrophone-options-exist-for-in-the-field-listening](https://bioacoustics.stackexchange.com/questions/1269/what-affordable-hydrophone-options-exist-for-in-the-field-listening)

Val''s current phantom-powered preamp circuit which is under development!!:
![image](https://user-images.githubusercontent.com/4945245/215250511-c8ff6c33-ec86-4298-91bd-01430c5dca38.png)

**Comment by bonnettde** [comment]
Hello Val and Scott
We will answer your specific questions and getback to to you soon.  Below is an preliminary outline of our Barn Project
[WetSound Project Outline .docx](https://github.com/orcasound/orcanode/files/10528133/WetSound.Project.Outline.docx)

**Comment by scottveirs** [comment]
Hey David, welcome to Orcasound! As we continue our discussion here, you might enjoy learning about [Markdown on Github](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax). To save folks without access to Word the trouble of downloading your Word doc and then opening it a compatible program, I used [Pandoc online](https://pandoc.org/try/) to convert your nice outline into Github markdown (and added headings):

# Wetsound Project

##Things to do

1.  Barn Presentation

2.  Acquisition System Design

    1.  Hydrophone

        1.  Research Hydrophone Technologies

        2.  Prototype development

            1.  Select hydrophone technology

            2.  Assemble hydrophone

            3.  Test protocol

            4.  Test Hydrophone

    2.  Acquisition System Design

        1.  Subsurface

            1.  Anchor System\`

                1.  Mooring Line

                    1.  Mooring Line Selection

                        1.  Requirements

                            1.  Support Buoy System

                            2.  Non-fouling

                        2.  Design Mooring Line

                2.  Anchor

                    1.  Selection

                        1.  Requirements

                            1.  Cannot Drag

                            2.  Easy to deploy and recover

                        2.  Design Anchor

            2.  Electronic Acquisition System

                1.  Raspberry Pi

                    1.  Program

                    2.  A/D conversion

                2.  Power Supply

                    1.  Battery Selection

                    2.  DC/DC converter 12VDC to 5 VDC

                    3.  Solar Controller

                3.  Configure/modify Raspberry Pi 

                    1.  Design and build send/receive antenna

                    2.  Data SD Card Memory

                4.  Bench Test System

                    1.  Program functionality

                    2.  EMI

                        1.  Resolve interference issues

        2.  Surface

            1.  Mooring Buoy

                1.  Buoy Selection

                    1.  Requirements

                        1.  Cylindrical

                        2.  Support 2 antennas

                        3.  Support Solar Panel

                        4.  Night light

                        5.  Buoy must remain on surface

                        6.  Long lifetime

                    2.  Design Buoy

        3.  Shoreside

            1.  Requirements

                1.  Computer System

                    1.  WiFi Capable

                    2.  Audio Output

                    3.  Wave File memory storage

                    4.  Operating System-Windows, Mac, Linux

                    5.  Ram-8GB

                    6.  Hard Drive 100GB

                2.  Software

                    1.  Audacity

                    2.  Advanced Analysis

                        1.  Raven

                        2.  Matlab

            2.  Raspberry Pi Code

                1.  Develop

                2.  Test

**Reply by bonnettde** [reply]
Thanks Val.  I'll try Pandoc

**Reply by scottveirs** [reply]
This is Scott, Val's son, actually!

**Comment by scottveirs** [comment]
@bonnettde We can definitely help with 2.i.a-b (Hydrophone R&D) and maybe the streaming/analysis side of things, **but** you will be helping us learn about anchors and buoys! 

Also, I might add one subsurface challenge: minimizing flow and other noise from the mooring system. I think there are some "tricks" for isolating the hydrophone acoustically, but the impacts of flow noise may depend mostly on where you deploy (Race Passage or Agate Pass harder than on east side of Bainbridge facing main basin of Puget Sound).

Possible leads:

- [ ] Study and/or ask how the [Jupiter Foundation](https://jupiterfoundation.org/listen) isolated the hydrophone on their buoyed systems off Puako (Kona coast of Big Island, Hawaii) and later on their waveglider prototypes. Their website has changed a lot over the years, so you'll have to dredge through using archive.org -- e.g. this [equipment page on "sonobuoy" design](https://web.archive.org/web/20060719212906/http://www.jupiterfoundation.org/new_bw_equipment.html). As I recall (from a brief site visit ~2004), they lashed a circular ring/loop of vibration absorbing / elastic material to the mooring line and then mounted hydrophone on the outer edge of the loop.
- [ ] Study [Sofar's Spotter buoy](https://www.sofarocean.com/products/spotter) and [Smart buoy](https://www.sofarocean.com/products/smart-mooring). 
- [ ] Talk to Jason Wood at SMRU, Ltd. in Friday Harbor about lessons learned from engineering their Coastal Acoustic Buoys (CABs)?
- [ ] talk to engineers at NOAA/PMEL, UW/APL/OOI about mooring technology in general, and specifically keeping hydrophones quiet on/near moorings
- [ ] Consider off-the-shelf components (like this [solar buoy](https://dunnriteproducts.com/solar-buoy-mooring/)) vs DIY-only solutions

**Comment by scottveirs** [comment]
For the record, from the Internet Archive, here's a version of the old Jupiter Foundation page on the equipment they used to listen for humpbacks back in ~2005... 

<img width="736" alt="Screenshot 2023-01-28 at 6 08 44 PM" src="https://user-images.githubusercontent.com/14044595/215301050-64fa57b0-04cc-4ba7-88c0-c727613fdce8.png">

I have no idea what they used to anchor the sonobuoy, nor how the hydrophone was connected to the transmitter. But in general, I'm very interested in exploring analog-only transmission to shore. If that was easy and fidelity wasn't lost in transmission, then the buoy engineering might be greatly simplified -- effectively moving most of the electronics from the subsurface/buoy to onshore...


![buoy_assembly_sm](https://user-images.githubusercontent.com/14044595/215304544-9d9c476b-b0a2-416c-a0ac-a543a7de6bf1.gif)

![New 2009 Buoy with Mauna Kea in bkgrd2_1](https://user-images.githubusercontent.com/14044595/215304696-0df36cc5-2a8e-49e1-b83d-d614f40926e1.jpg)

**Comment by bonnettde** [comment]
Hi Scott
Thanks for this info.  We are hoping to be able to design an in water
package with a smaller footprint than Jupiter and do the A-D onboard then
transmit via wireless to shore.  We also will use a Li Ion battery and have
sufficient on board storage for .wav files to be held until the user wants
to download them.  Of course this is all in the preliminary stage of
concept design so any info like Jupiter is welcomed.

**Reply by scottveirs** [reply]
Sounds, good. Two thoughts:

1. Why transmit to shore and also store onboard? (Val mentioned you are interested in studying SRKW clicks so focused on 192kHz sampling rate minimum, so maybe you intend to store full lossless data onboard and transmit lower resolution, lossy data to shore?)
2. If you're going to store onboard, you might consider [FLAC](https://xiph.org/flac/) or [X3](https://github.com/psiphi75/x3-spec) lossless compression which are more efficient than WAV. Orcasound currently streams lossy in HLS (but would like to try DASH at some point) and archives in FLAC (so far 48kHz max, but the Pisound ADC in [Orcasound's current open hard/soft-ware solution](https://www.orcasound.net/2018/04/27/orcasounds-new-live-audio-solution-from-hydrophone-to-headphone-with-a-raspberry-pi-computer-and-hls-dash-streaming-software/) can do 192kHz at 24 bits for 2 channels).

**Reply by bonnettde** [reply]
We are currently hoping the Raspberry Pi Zero 2W (https://datasheets.raspberrypi.com/rpizero2/raspberry-pi-zero-2-w-product-brief.pdf ) can be set up to accomplish both real time (near) audio to shore and to store .wav files that can be analyzed later.  The 2W has a SD sound card slot USB and uses both Bluetooth and 802.11 wireless.  With appropriate programming we hope we can set sample rates for A/D at up to 500KHz.  Our first priority is to develop a hydrophone that is affordable and has an appropriate sensitivity profile to allow fairly high sample rates.  I like your Orcasound solution and will look it over.
Thans for this information
Dave

**Comment by scottveirs** [comment]
@dbainj1 This is another place to try to push forward on an Orcasound buoyed solution!

**Comment by scottveirs** [comment]
@bonnettde What radio technology are y'all considering for your first prototype?

For the [Sunset Bay hydrophone site](https://live.orcasound.net/sunset-bay) we might get by with one of the wireless mics used for sports and entertainment events. One option is the Rode Go II wireless mic that claims to have a range of about 250 meters. [Glazer's  Rentals](https://www.glazerscamera.com/pages/rentals) in Seattle will rent one of these (for testing) at a rate of $20/day. 

Other options they rent for are made by DJI and Sennheiser

**Comment by scottveirs** [comment]
Hey @bonnettde and @dbainj1 -- have you all seen this buoy/drifter project for real time classification of underwater noise from vessel?
[https://github.com/IQTLabs/AISonobuoy#particletrackerbuoy](https://github.com/IQTLabs/AISonobuoy#particletrackerbuoy)?

Looks like some interesting components and DIY ideas (e.g. PVC pipe for waterproof housing)...

**Reply by scottveirs** [reply]
Looks like they had two configurations for their AIS sonobuoy concept:

1. A drifting sonobuoy (w/global option) design -- a Adafruit FeatherWing coupling an Aquarian hydrophone to an evaluation board... all within a ballasted Schedule 40 PVC pipe:

<img width="1277" height="2054" alt="image" src="https://github.com/user-attachments/assets/15cb23fe-66f1-493a-9e0f-885051dffec1" />

Maybe the most useful bits for Orcasound are the [chassis style files](https://github.com/IQTLabs/AISonobuoy/blob/main/ParticleTrackerBuoy/README.md#hardware) and waterproof methods for integrating the Aquarian hydrophone with the PVC pipe...

<img width="1277" height="2054" alt="image" src="https://github.com/user-attachments/assets/e760112f-2e10-4299-a555-666b7e2d103d" />

It's not clear that they did more than design this version. I can't find any imagery evidencing a build or deployment...

2. And an (earlier?) coastal case design -- a Raspberry Pi Zero (2W) with a HiFiBerry ADC Pro in a waterproof case that could go on any platform within 4G wireless coverage -- documented in the [aisonobuoy-collector-raspberrypi repo](https://github.com/IQTLabs/aisonobuoy-collector-raspberrypi): 

<img width="1280" height="1024" alt="image" src="https://github.com/user-attachments/assets/46ee4670-8350-463c-b88d-fe46335b9dfa" />

The modular case idea came from the current tracking [makerbuoy project](https://www.makerbuoy.com/products-1) (GPS + Arduino + Iridium)... from Johns Hopkins APL in 2018:

<img width="1285" height="1686" alt="image" src="https://github.com/user-attachments/assets/64a0f19e-8166-4436-af3b-5a917a5ce8dd" />

**Comment by scottveirs** [comment]
On the ride home from our first visit to the BARN, @dbainj1 Dave mentioned long ago using a pager to turn on/off a buoyed acoustic project. It looks like there is a similar low bitrate (e.g. texting or walkie talkie) technology called LoRa that might be of interest — https://en.m.wikipedia.org/wiki/LoRa — e.g. for waking up a buoy?   

Not sure they will deliver the promised 5 km messaging and 300 meter “crystal clear” audio transmission, but this active MessengerPi Kickstarter looks interesting! https://www.kickstarter.com/projects/diytech/messengerpi-lora-based-messenger-and-walkie-talkie

**Comment by klaplain** [comment]
If we assume that we need to accurately transmit sounds with 16 bit resolution at frequencies up to 20kHz, what distance does the WiFi need to reach?   Is it line of sight?  There's a big difference between 300m and 5km.  Lora is viable for low bandwidth pathways but I don't think it is suitable for the real-time transmission of high resolution sound

**Comment by dbainj1** [comment]
The distance needed will vary with site.  About 500', give or take, would be a good initial goal.  Line of sight can typically be assumed.  When line of sight isn't available, it's likely that relays could be set up that allow line of sight.We've looked at FM radio, which requires a license to go over 200'.  Bluetooth is another option, but probably won't reach 500'.  Wi-Fi is an option that's good for miles, if you can keep the antennas lined up.  Wide band FM is another technology that could go to higher frequencies.  A mobile hotspot can work where there is good service.  Satellite Internet is another option, although expensive and typically limited to less than 3 Mb per second.Of course, if you have a local resident who just needs to get to the end of the dock, the distance could be far shorter.  We've had one application where we needed to connect Wi-Fi a few miles away (it worked most, but not all the time).Radar is a possible source of interference at Wi-Fi frequencies.I'd suggest developing a system that uses technology you're comfortable working with.  We'd apply it where it can work.  If you develop longer range solutions in the future, we can apply them at more challenging sites.DaveSent from Samsung Galaxy smartphone.
-------- Original message --------From: klaplain ***@***.***> Date: 11/22/23  1:35 PM  (GMT-08:00) To: orcasound/orcanode ***@***.***> Cc: dbainj1 ***@***.***>, Mention ***@***.***> Subject: Re: [orcasound/orcanode] Can Orcasound imagine a buoy-based low-cost
  live-listening hydrophone system for all orca signals? (Discussion #59) 
If we assume that we need to accurately transmit sounds with 16 bit resolution at frequencies up to 20kHz, what distance does the WiFi need to reach?   Is it line of sight?

**Reply by scottveirs** [reply]
@klaplain yes I agree with @dbainj1 that a great initial design goal for a wireless orca buoy would be:

- Resolve frequencies up to 20 kHz (could be relaxed to 8-10 kHz IMHO)
- 16 bit depth
- 150m (~500') range to buoy from streaming computer ashore

**Comment by scottveirs** [comment]
A Google search yesterday revealed this interesting [effort to build a smart buoy for Orcasound](https://www.hackster.io/MazerRuckus/spresense-smart-buoy-e8c6f7)! (I wasn't aware of this til now.)

I'm new to cellular IoT communication schemes, but it appears the [chip](https://altair.sony-semicon.com/products/alt1250/) on the LTE extension board may support:

"CAT-M1 R14 with DL/UL speeds of 588/1119 Kbps"

Especially with some lossy compression, it seems that upload bandwidth might be sufficient...

**Reply by klaplain** [reply]
Yep.  For distances of a few 100 feet, WiFi/Bluetooth would be OK.  But for longer distances LTE is a better solution.  Data rates should support the 16bit/20kHz you need.  There would be a subscription cost for the LTE data link (estimate $100/year).  The Spresense Sony board is a great way to test for viability.  An alternative BARN strategy would be to reuse the existing BARN preamplifier & STM32 ADC and to pair this with a board such as the Adafruit FONA 3G Cellular board. (https://www.adafruit.com/product/2687)

**Comment by scottveirs** [comment]
@dthaler This is a good thread for adding ideas regarding transmission to land from a future buoyed solution...

**Reply by dthaler** [reply]
https://www.amazon.com/Wireless-Bridge-AdaLov-Multipoint-Outdoor/dp/B0BKSM4NV1/ref=sr_1_3_pp is an example of a wireless bridge with a range of like 2 miles, though it has a 60 degree directional window so requires it to stay approximately pointed in a general direction.

**Reply by scottveirs** [reply]
Keeping a buoy oriented in the right direction sounds tricky, but a spike in the seafloor would do the job, or attachment to an existing subtidal piling... 

As of early 2026, the cost of this bridge is just over $200 (US).

**Comment by dbainj1** [comment]
Dave,

I'm familiar with this technology.  We used it to transmit signals from Point Robinson across the sound to Des Moines for a while.

One of the issues is power.  If this is provided by POE, that defeats the purpose of going wireless.  However, if we use solar, wireless would be beneficial.

We ran into a couple problems using it.  One, it needs to be line of sight, and that was blocked at high tide (this won't be a problem at the new site I mentioned).  The other was throughput.  It dropped off at times of day when lots of other people were using wireless (less likely to be a problem at the new site).  Radar from passing ships also seemed to interfere with the signal, since radar and wi-fi use the same frequency.

When it worked well, it was far superior to the wi-fi hotspot we switched to.  However, the project manager opted to stick with the hotspot due to the intermittent nature of cross-sound wi-fi as we implemented it.  I would have tried to move the antennas to maintain line of sight at high tide, and hoped that that would have limited the other problems.

Anyway, the new site is unusual in that the protected marine vegetation is fairly limited.  The flat terrain means you could walk out to the buoy to tend it at low tide, or reach it in a boat at high tide.  A wi-fi bridge is definitely an option, whether it's pointed at a neighbor's house or across the sound.

--Dave

________________________________

**Comment by scottveirs** [comment]
Interesting Blue IQ OpenEar project that's integrating hydrophone in solar-powered buoys on eastern seaboard (mostly for NARW detection?) -- [https://oceannews.com/frontline/check-the-tech/redefining-acoustic-monitoring-an-exclusive-look-at-blueiqs-openear-hydrophone/](https://oceannews.com/frontline/check-the-tech/redefining-acoustic-monitoring-an-exclusive-look-at-blueiqs-openear-hydrophone/)

**Comment by scottveirs** [comment]
From @dbainj1 back on September email -- 

On Thu, Sep 12, 2024 at 22:40 David Bain <dbain@orcaconservancy.org> wrote:
Scott,

I've been looking into communications from a buoy.

I've got two preferred candidates.

My first choice is wi-fi.  To get extra range, I think we could attach an omni-directional antenna to the Pi with 9 dB of gain.  We could put a directional antenna on the computer on shore.  The gain for the onshore antenna will depend on its directionality.  The beam shape we want will depend on how far the buoy can move.  I.e., the mooring line will need to be long enough to get to the surface at high tide plus wave height, which will give it a lot of slack to move around at low tide.   It will be easy enough to do the trigonometry to figure out how wide the vertical and horizontal beams need to be.  

If the antennas don't provide enough gain for the distance, we could go to powered wireless bridges, but they'll require extra power on the buoy.

The other approach is to go 5G.  We'd probably need a wireless home internet plan, which would cost more than piggybacking on an existing internet plan.  That would probably increase power requirements by a couple Watts.  But, we'd be independent of shore infrastructure.  I'll need to look into whether we can get away with a cheap cell phone, or we'd need a more expensive hot spot device.  From what I've found so far, it seems like hotspots can handle more devices, which is a feature we don't need, so a phone might be fine.

FM didn't make the cut because of licensing requirements for going more than 200'.

Satellite would probably be limited to 2-3 Mbs, which would probably be ok for lossy audio that's good enough for OrcaHello and Orcasound, but probably not if we want 24 bits at 192kHz with lossless compression.  Similarly, we probably won't get the bandwidth we'd like to have with Bluetooth, but we might get enough.

As for power, a Pi wants 15 watts.  That's 360 Whr/day.  A 100W solar panel should provide an average of 500Whr/day, which would be enough, but I'm guessing we'll need to supplement power over the winter.  A battery the size of my bike battery would get us through a day,  But, we'd be looking at maybe 250 AH for a 12V battery to be sure we'd make it through a week without any supplemental power.

Windmills that produce 400-800W are reasonably sized, but won't produce much, if any power, when the air is relatively still.

Tidal power could be pretty reliable, but would probably foul over time, so wouldlikely be high maintenance.

I haven't come across any small-scale wave power generators.  That could be a project for BARN.

Thermal power would be interesting if we could pull it off.  Stirling engines can generate power with pretty low temperature differences, so if the computer gets up to 50 C and sea water is around 10 C, we could probably recover some energy that way.

I'll get back to you separately with thoughts on the non-electrical parts of the buoy.

--Dave

**Reply by scottveirs** [reply]
@dbainj1 it will be interesting to compare your wireless range tests with these results from Jim Henderson today on the BARN Slack channel: 

![image](https://github.com/user-attachments/assets/2494851e-8395-4c78-bc48-3bf91f2c72ce)

When you get an order of buoy parts organized, could you document here what you procure (in terms of makes/models of hardware)?

**Comment by scottveirs** [comment]
Here's a new source of tidal power to keep an eye on -- [rotation from vortex induced vibration](https://www.witt-energy.com/howitworks)

 It may only generate ~5W in typical Puget Sound currents and currently costs ~30k$, but the idea of power from tidal currents without any external moving parts is intriguing!

**Comment by scottveirs** [comment]
Hey @dbainj1 and @veirs -- 

Here's a brainstorm sketch I made when thinking about deployments on the outer edge of Puget Sound, e.g. off of Eglon to monitor the west side of the "Possession Triangle." In many prospective sites, it could leverage an existing charted piling, or if there is no "piling or dock end of convenience), it could be a wand that gets screw-anchored into the sediment.

The design is mainly motivated by trying to minimize flow noise that often plagues buoys, while still getting the hydrophone deep enough for useful detection range and maintaining orientation of a surface antenna...

![IMG_3214](https://github.com/user-attachments/assets/4272b3ed-aee2-434c-9323-b54c0fdfaabb)

What do you (all) think?

**Reply by scottveirs** [reply]
We could call it an "ear spear" or a "spike mic" or ??

**Comment by dbainj1** [comment]
Scott,

I completed my (soccer) field test of the electronics for a nearshore buoy.

Power:  Nicesolar 50W bifacial solar panel with charge regulator          $50
                 Fukuai 12V 16 Ah lithium iron phosphate battery                          $40

Transmitter-receiver pair:  1Mii wireless audio (2.4 GHz, 24-bit, 192 kHz)  $60

The transmitter-receiver pair worked at 100m as advertised.  The frequency response was good from 60 Hz to 22 kHz.  The power source is a 1A USB AC power adapter, which can provide up to 5W.  Actual power usage may be less.

At 5W, it would use 120Wh per day.  A 50W solar panel should provide 250Wh per day on average.  However, we can expect sub-optimal orientation and short days in winter, so actual charging in winter could be significantly less.  Bifacial panels are supposed to generate power with light from both sides, so I'm hoping that reflected light off the water will generate some power.

The charge regulator has USB out, which works in the absence of panel input, so it should run the transmitter off the battery at night and recharge the battery during the day.

We still need to design and build the mechanical part (mooring, floats, waterproof housing, etc.) and test it under marine conditions where the antenna moves around and changes orientation, but it should work as an "ear spear."

If I make it to the BARN tomorrow, I'll bring the parts with me.

--Dave

________________________________

**Comment by scottveirs** [comment]
Nice story about Matt Pine's live buoy in New Zealand -- 
https://liveocean.org/project/buoys-that-listen/

![Screenshot 2025-02-19 at 12 30 06 PM](https://github.com/user-attachments/assets/64a2a567-af35-4b08-b176-6ef3df1670a4)

**Reply by scottveirs** [reply]
@dbainj1 Interesting to see how they handled smaller solar panels at a suite of azimuths and optimal sun angles within a glass sphere!

**Comment by scottveirs** [comment]
Here's another small float-based acoustic real time solution (from Nova Scotia) but lots of batteries instead of solar panels -- 
https://turbulentresearch.com/products/tr-float

![Screenshot 2026-02-05 at 9 06 48 AM](https://github.com/user-attachments/assets/da9bdad2-d174-4b6a-8e54-84667e7ee2aa)

**Comment by scottveirs** [comment]
The Spotter buoys are starting to look pretty attractive (for ~$20k)!
https://www.sofarocean.com/products/spotter/solutions/sound

![Screenshot 2025-07-15 at 10 15 09 AM](https://github.com/user-attachments/assets/1dc97753-120e-4add-a25a-9501ce5071ba)

**Reply by scottveirs** [reply]
Seasounds project is using this in 2025-2026 to prospect for whale sounds in SF Bay -- 

https://www.sofarocean.com/posts/bayquest-seasounds-project

![Screenshot 2026-02-05 at 9 10 51 AM](https://github.com/user-attachments/assets/802ac22d-b4d3-4541-85f4-e58f3ea9e550)

**Comment by scottveirs** [comment]
Hey @dbainj1 can you post make/model &/or specs for your wireless device? I'm particularly interested in its power draw as I talk live buoys with Matt Pine in New Zealand.

**Comment by dbainj1** [comment]
Scott,

It's a

1Mii 2.4Ghz (1 TX + 1 RX) Wireless Audio Transmitter Receiver for TV, 320ft Long Range 20ms Low Delay HiFi Audio, Wireless Adapter Kit for Powered Speaker/Stereo/Soundbar, RCA(3.5mm AUX) Out/Input

More details are available at:

https://www.amazon.com/dp/B09MCGQ8S2/ref=sspa_dk_detail_6?pd_rd_i=B09MCGQ8S2&pd_rd_w=N19Ca&content-id=amzn1.sym.f2f1cf8f-cab4-44dc-82ba-0ca811fb90cc&pf_rd_p=f2f1cf8f-cab4-44dc-82ba-0ca811fb90cc&pf_rd_r=C5GXTD6P65X6YNCNDY22&pd_rd_wg=eKsr1&pd_rd_r=0edbf69c-858d-46e4-8e2b-1a5a7cd8c907&sp_csd=d2lkZ2V0TmFtZT1zcF9kZXRhaWxfdGhlbWF0aWM&th=1

It runs off a 5W USB power source, but it may not use all of that.

--Dave
________________________________

**Comment by scottveirs** [comment]
Old note from Steve Kahle via Jim Luby email: [Steminc](https://www.steminc.com/) is an affordable source for piezo material. 

And here’s a video recommended by Jim regarding the [Gladys hydrophone](https://m.youtube.com/watch?v=yV5F_CjixDg&pp=0gcJCWQCo7VqN5tD) instructable with lots of circuitry details.
